### DISCUSSION TITLE
Visualisation du modèle de données

### DISCUSSION THREAD

**Discussion by GoodFeel91** [dbarzin/mercator#237] [Q&A]
Bonjour,

Après avoir rempli un certain nombre d'éléments sur ma cartographie, je souhaiterais afficher le modèle de données. Or, c'est toujours celui par défaut qui s'affiche et je ne vois pas où puis-je générer mon modèle de données basé sur ma cartographie :

![image](https://user-images.githubusercontent.com/78422494/195652635-c22822eb-8628-4c91-9b85-01b247d7cc00.png)

Comment pourrais-je générer mon modèle de données ?

Merci par avance de votre aide.

**Comment by dbarzin** [comment]
C'est normal, ce graphique représente tout les liens qu'il est possible de définir.
Pour voir ta cartographie, tu peux afficher un objet par exemple une application et cliquer sur le bouton "explorer".
Chaque fois que tu double-click sur un objet, cela ajoute tout les objets qui y sont liés.

![image](https://user-images.githubusercontent.com/43208232/195679814-5e66eb92-4c1e-4656-a466-b88d837227aa.png)

**Reply by GoodFeel91** [reply]
Ok je comprends mieux.

Effectivement, la cartographie se déroule dans la partie "Explorer". Je pensais que la présentation était identique à l'exemple de modèle de données, notamment sur la représentation en silo par vue (Infrastructure, Entité, etc).

Merci pour ce complément d'information.
